server/src/server.js

```diff
@@ -256,7 +256,8 @@ app.use(function (req, res, next) {
     }
   }
   req.abTests = authInfo.abTests || {};
-  req.backend = `${req.protocol}://${req.headers.host}`;
+  const host = req.headers.host === config.contentOrigin ? config.contentOrigin : config.siteOrigin;
```

Comment:
Might be nice to log cases when we get an unusual host header, otherwise looks good